core/modules/ckeditor5/ckeditor5.admin.inc

```diff
@@ -196,7 +196,7 @@ function _ckeditor5_settings_parse_style_list($style_list_string) {
  * @return string
  *   The string representation of the style list.
  */
-function _ckeditor5_settings_stringify_style_list(array $style_list = array()) {
+function _ckeditor5_settings_stringify_style_list(array $style_list = NULL) {
```

Comment:
I don't think we should be changing the default value for $style_list here.

Reply:
Yes, it was a mistake.